### PR TITLE
Renamed identifier to id

### LINKED ISSUE #2351: Rename `identifier` to `id` in `/assets/homepage_images/image_info.json`
## Problem

Throughout the frontend code, we use the name `id` for the Openverse UUID of a media item. However, in the Homepage Gallery, this property is confusingly called `identifier`.

## Description

Rename the `identifier` field in `frontend/assets/homepage_images/image_info.json` to `id`.
Rename `identifier` instances in `VHomeGallery.vue` component to `id`.
Rename `identifier` to `id` in `analytics.ts`

## Alternatives

Rename all occurrences to `identifier` to match the catalog - however, this would require a lot of effort in the API>

## Additional context

This came up when reviewing #2336

### PR DESCRIPTION
**Pull request by duwarakan**
## Fixes
Renamed identifier to id.

Fixes #2351

## Description
Renamed identifier to id in `frontend/assets/homepage_images/image_info.json`, `VHomeGallery.vue`, and `analytics.ts`.

### CODE REVIEW

File: frontend/src/types/analytics.ts [hunk 1]
```diff
@@ -37,8 +37,8 @@ export type Events = {
   CLICK_HOME_GALLERY_IMAGE: {
     /** the set to which the image belongs */
     set: string
-    /** the identifier of the image */
-    identifier: string
+    /** the id of the image */
```
Comment (obulat) [inline frontend/src/types/analytics.ts:40]:
```suggestion
    /** The unique ID of the media */
```

**Review by obulat — APPROVED** [review]
Awesome, @duwarakan 🎉  Thank you for fixing this issue.
I've added a suggestion to change the comment to match the other events' comments, other than that, it's ready to go 🚀

File: frontend/src/types/analytics.ts (hunk 1 above)
Comment (duwarakan) [reply frontend/src/types/analytics.ts]: Thanks for encouraging me. Suggested changes have been applied.

**Comment by obulat** [comment]
Noting that the CI failure is unrelated, and should be fixed by #2379

**Review by dhruvkb — APPROVED** [review]
LGTM 👌

**Pull request merged**